### PR TITLE
Add desktop Chrome

### PR DESCRIPTION
**Pull request by bebef1987**
I needed to open chrome and this code helped

### CODE REVIEW

File: apps/desktop/chrome.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from selenium import webdriver
+from selenium.webdriver import Chrome
+
+
+class DesktopChrome(object):
+    def __init__(self, proxy, *args):
+        self.proxy = proxy
+
+    def start(self, url="about:blank"):
+        print("START")
```
Comment (davehunt) [inline apps/desktop/chrome.py]: nit: can we remove this print statement?

Comment (davehunt) [reply apps/desktop/chrome.py]: I just noticed we have this in `firefox.py` can you remove it from both?

File: apps/desktop/chrome.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+        chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument('--proxy-server=127.0.0.1:8080')
+        chrome_options.add_argument('--proxy-bypass-list=localhost;127.0.0.1')
+        chrome_options.add_argument('--ignore-certificate-errors')
+        chrome_options.add_argument('--no-default-browser-check')
+
+        driver = Chrome(chrome_options=chrome_options)
```
Comment (davehunt) [inline apps/desktop/chrome.py]: Let's rename `chrome_options` to `options` as it's supported as a kwarg and more succinct.

File: studio.py [hunk 3]
```diff
@@ -1,10 +1,11 @@
+from __future__ import absolute_import, print_function
```
Comment (davehunt) [inline studio.py]: Is this necessary? I notice you've used relative imports elsewhere. Can we be consistent throughout. I'm not sure I have a preference between relative and absolute.

**Review by davehunt — CHANGES_REQUESTED** [review]
Thanks, it's good to support Chrome, too! :)

File: apps/desktop/chrome.py (hunk 1 above)
Comment (bebef1987) [reply apps/desktop/chrome.py]: done

File: apps/desktop/chrome.py (hunk 2 above)
Comment (bebef1987) [reply apps/desktop/chrome.py]: done

File: studio.py (hunk 3 above)
Comment (bebef1987) [reply studio.py]: Removed

**Comment by bebef1987** [comment]
Updated

**Review by davehunt — APPROVED** [review]

**Pull request merged**